bingo/symbolic_regression/agraph/mutation.py

```diff
@@ -91,6 +99,14 @@ def __call__(self, parent):
         mutation_algorithm = self._mutation_function_pmf.draw_sample()
         mutation_algorithm(child)
 
+        for i, (command, op1, op2) in enumerate(child.command_array):
+            if not IS_TERMINAL_MAP[command]:
+                if op1 >= i or op2 >= i:
+                    print("bad!", self._last_mutation_type, self._last_mutation_location)
+                    print("parent", parent.command_array)
+                    print("child", child.command_array)
+                    raise(RuntimeError)
+
```

Comment:
Is this chunk of code meant for only debugging? or are we intending to keep it here?  If we need to keep it, i think we can make it more efficient, i.e., vectorizing rather than the loop/

Reply:
Ah yes that is my bad, this chunk of code is only used for debugging; it was in the old code so I forgot to remove it, thanks for catching that.